clientv3/integration/kv_test.go

```diff
@@ -547,3 +547,13 @@ func TestKVGetCancel(t *testing.T) {
 		t.Fatalf("cancel on get broke client connection")
 	}
 }
+
+// TestKVPutStoppedServerAndClose ensures closing after a failed Put works.
+func TestKVPutStoppedServerAndClose(t *testing.T) {
+	t.Parallel()
+	defer testutil.AfterTest(t)
+	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+	clus.Members[0].Stop(t)
+	clus.Client(0).Put(context.TODO(), "abc", "123")
```

Comment:
Add the comment to tell that this test depends on afterTest to check leak? Or it is unclear to me why we just do a put without checking anything.